docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,259 @@
+
+Additionally, a visual component is rendered in the step **Configuration** to accept this input:
+
+<img src="./images/custom-string-prop.png" alt="Custom props render in the Configuration portion of the code step.">
+
+### Connecting apps
+
+In the v2 builder, you can connect apps with your code using [props](/components/props).
+
+Above the `run` function define an app prop that your Node.js step integrates with.
+
+```javascript
+export default defineComponent({
+  props: {
+    twitter: {
+      type: "app",
+      app: "slack",
+    }
+  },
+  async run({ steps, $ }) {
+    // now your Slack token for authenticating API calls will be available here:
+    this.slack.$auth.oauth_access_token;
+  },
+})
+```
+
+After testing the step the Slack app will appear in the **Configuration** section on the left hand side. In this section you can choose which Slack account you'd like to use in the step.
+
+<img src="./images/app-props-example.png" alt="Example of adding an app connection to a v2 Node.js step">
+
+
+### HTTPs Response
+
+You can still return an HTTP response from an HTTP triggered workflow.
+
+Use `$.respond` to send a JSON or string response from the HTTP call that triggered the workflow.
+
+```javascript
+export default defineComponent({
+  async run({ steps, $ }) {
+    $.respond({
+      status: 200,
+      headers: {},
+      body: { 
+        message: "hello world!"
+      }
+    });
+  },
+})
+```
+
+Please note, you'll also need to configure the HTTP trigger step to also allow custom responses. Use the dropdown in the **HTTP Response** section of the HTTP trigger to select the **Return a custom response from your workflow** option.
+
+<img src="./images/custom-http-response-option.png" alt="Select the option to allow your workflow to send it's own HTTP responses">
+
+## Known Gaps & Limitations
+
+The new workflow builder introduces new features that weren't available in v1. However there are some features from the original builder that are not available in v2 currently.
+
+### Sharing workflows
+
+At this time, sharing is not yet implemented in v2 of the workflow builder.
+
+We're working on bringing this same feature to the new version, if you need assistance transferring workflows across accounts, [please contact us](/docs/support).
+
+### `$checkpoint`
+
+The `$checkpoint` functionality to save data between workflow runs has been removed.
+
+But you can leverage the `$.database` service to store arbitrary data across your workflow runs like unique IDs. [Read more about using `$.database` here.](https://pipedream.com/docs/code/nodejs/#managing-state)
+
+::: warning
+Please note that any values stored in `$.database` are only accessible in subsequent workflow runs _in the same step_.
+:::
+
+### Public workflows
+
+At this time, all v2 workflows are private.
+
+Unlike v1, it's not possible to make a workflow public. We can still help support your workflow, you just need to share it with us under the workflow settings.
```

Comment:
"We can still help support your workflow, you just need to share it with us under the workflow settings." -> "If you're working with Pipedream support to troubleshoot your workflow, you can share it with the support team under your workflow's Settings"